src/services/administration/routes.ts

```diff
@@ -60,9 +61,24 @@ const administrationRoutes: RouteConfig = {
                 },
                 {
                     path: 'policy',
-                    name: ADMINISTRATION_ROUTE.IAM.POLICY._NAME,
-                    meta: { lnbVisible: true, label: getMenuLabel(MENU_ID.ADMINISTRATION_POLICY) },
-                    component: PolicyPage as any,
+                    meta: { label: getMenuLabel(MENU_ID.ADMINISTRATION_POLICY) },
+                    component: { template: '<router-view />' },
+                    children: [
+                        {
+                            path: '/',
+                            name: ADMINISTRATION_ROUTE.IAM.POLICY._NAME,
+                            meta: { lnbVisible: true },
+                            props: true,
+                            component: PolicyPage as any,
```

Comment:
```suggestion
                            component: PolicyPage,
```

PolicyPage 컴포넌트를 `defineComponent` 함수로 감싸면 타입에러가 안날거에요~

Reply:
요거 성모님이 잘 알고계십니다!